app/assets/stylesheets/desktop/topic.scss

```diff
@@ -77,13 +77,23 @@
 
 #topic-progress-wrapper {
   position: fixed;
-  right: 1em;
+  right: 2em;
   bottom: 0;
+  left: 0;
+  margin: 0 auto;
+  max-width: $reply-area-max-width;
+  display: flex;
+  justify-content: flex-end;
```

Comment:
Just a note for anyone looking at this:
it will display the progress wrapper at `(screen_width - $reply-area-max-width) / 2 + 2em` from the right edge of the screen. 😉 (the combination of flex, max-width, margin auto, left and right wasn't immediately obvious to me, had to do some in-browser math 😛)

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/66961/92750659-98ec9f00-f387-11ea-88d4-e3f27e6f689a.png">